src/models/subscription.ts

```diff
@@ -198,10 +198,10 @@ export class Subscription extends Model {
 				gitHubInstallationId: payload.installationId,
 				jiraHost: payload.host,
 				jiraClientKey: payload.hashedClientKey,
-				gitHubAppId: payload.gitHubAppId || null,
-				avatarUrl: payload.avatarUrl || null
+				gitHubAppId: payload.gitHubAppId || null
 			},
 			defaults: {
+				avatarUrl: payload.avatarUrl || null,
```

Comment:
Do we even use it?

Reply:
If not, should we rather drop the column?

Reply:
It is a new column, added by xxxx team. :) I forgot the name.

Reply:
https://github.com/atlassian/github-for-jira/pull/2250/files#r1260508317